Mage.Client/src/main/java/mage/client/dialog/PreferencesDialog.java

```diff
@@ -3512,7 +3512,6 @@ private static void loadImagesSettings(Preferences prefs) {
             dialog.txtImageFolderPath.setText(path);
             updateCache(KEY_CARD_IMAGES_PATH, path);
         }
-        load(prefs, dialog.cbSaveToZipFiles, KEY_CARD_IMAGES_SAVE_TO_ZIP, "true");
```

Comment:
Do not delete loading code

Reply:
Thanks, now I get it, that needs a second parameter which is the default value: `load(prefs, dialog.cbSaveToZipFiles, KEY_CARD_IMAGES_SAVE_TO_ZIP, "true", "false");`

My change just happened to work since the reading from cache has also a default value in case the property is missing. 

I'll add the second parameter.